### PR TITLE
Fixes missing gradient colours in Marble banner in mobile size

### LINKED ISSUE #136: Marble banner title is white on smallest screen size

### PR DESCRIPTION
**Pull request by alexandercyu**
Added gradient colours for Marble banner in mobile size. Fixes #136

### CODE REVIEW

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
This same code is repeated in all media widths. Please just define it once instead of repeating it in multiple places.

**Comment by alexandercyu** [comment]
> This same code is repeated in all media widths. Please just define it once instead of repeating it in multiple places.

Removed repeated versions of the same code and defined it in one place.

**Review by ShrutiK100 — APPROVED** [review]
Looks good!

File: static/css/marble.css [hunk 1]
```diff
@@ -1393,6 +1387,76 @@ h6{
         margin-right: 10px
     }
 
+    /*About*/
+    .margin-committee-title{
+        margin-bottom: 30px;
+    }
+}
+
+@media (width >= 576px) {
```
Comment (mishaschwartz) [inline static/css/marble.css:1396]: Doesn't this just add back the extra `@media (width >= 576px)` block that you condensed into one in the other PR?

Comment (alexandercyu) [reply static/css/marble.css]: Pulled in the >=576px media query changes from fix-node-menu.

**Review by mishaschwartz — COMMENTED** [review]
Ok ... I have a few questions about this:

I'm a little confused why we need this in `width >= 576px`, `width <= 992px` and `width > 992px`. Can't we just move this out of the media queries entirely since these cover all cases?

Oh, I see that `width <= 992px` has the drop shadow and the others don't. Is that intended? Don't we want the drop shadow everywhere?

**Comment by alexandercyu** [comment]
> Oh, I see that `width <= 992px` has the drop shadow and the others don't. Is that intended? Don't we want the drop shadow everywhere?

No, the shadow is only at some widths.

**Comment by mishaschwartz** [comment]
> No, the shadow is only at some widths.

Are we sure that's what we want? That seems like an oversight to me. Did you and @ShrutiK100 discuss this

**Comment by alexandercyu** [comment]
> > No, the shadow is only at some widths.
> 
> Are we sure that's what we want? That seems like an oversight to me. Did you and @ShrutiK100 discuss this

Yes.  She made a point to say the title at some widths is "flat" and the quickstart menu only has a shadow on desktop because of the video.  The shadow for the Marble title may be the same.

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**